### ISSUE TITLE
Add support for selene

### ISSUE THREAD

**Issue by hackerjon49** [openandroidinstaller-dev/openandroidinstaller#301]
**Device informations**
- Brand: Xiaomi 
- Phone name: selene
- Device code (if unknown, leave blank): 211211198G
- Model (if several models exist): redmi 10 2022 / MIUI version MIUI Global 14.0.1 14.0.1.0 (TKUTRXM)

**Device images**  
Please provide links to any available images of the device, such as official stock firmware or custom ROMs you would like to install.  

**Do you own the device and would be willing to test the installer?**
- [y ]

**Additional context**  
Please provide any additional context or information that may be helpful in adding support for this device.

**Comment by MagicLike** [comment]
tagging @anon1892

**Comment by anon1892** [comment]
I didn't find any ROM for this device, and there is no XDA topic for this device...
You will have to use GSI, but 
1. The device is not referenced here https://github.com/phhusson/treble_experimentations/wiki/Xiaomi
2. Don't think OAI supports flashing GSI for now... Maybe I'm wrong?

**Comment by MagicLike** [comment]
> I didn't find any ROM for this device, and there is no XDA topic for this device... You will have to use GSI, but
> 
>     1. The device is not referenced here https://github.com/phhusson/treble_experimentations/wiki/Xiaomi
> 
>     2. Don't think OAI supports flashing GSI for now... Maybe I'm wrong?

hmmm :thinking:

**Comment by tsterbak** [comment]
Yeah, there is neither a TWRP nor a XDA-thread, so unlikely this will be supported soon.
GSI is out of scope for now, I think.
